src/datareaders/lucid/lucid_reader.py

```diff
@@ -38,11 +38,12 @@ def add_points(self):
             # Get the name, the building name (sometimes the same thing) and the units information
             try:
                 name, building_name, description = point_names[i].split(" - ")
+                name = name + " - " + "(".join(description.split("(")[:-1])
             except ValueError:
-                name = building_name + " - " + description.split("(")[0]
-
-            if "Old Meter" in description:  # We need to differentiate between old meters and new ones.
-                name = name + "(Old Meter)"
+                building_name, description = point_names[i].split(" - ")
+                name = building_name + " - " + "(".join(description.split("(")[:-1]) # remove units information
```

Comment:
How is this removing units information? What does description look like before this line?

Reply:
Is it just separating units and then we ignore parentheses parsing for units?

Reply:
Description might look like "Steam (kBTU)" or "Steam (Old Meter) (kBTU)"

Reply:
I see! Thanks